i18n/en/design-system.json

```diff
@@ -43,5 +43,32 @@
   "global-footer-licensing-and-vertical-description-param-vertical-music": "Music",
   "global-footer-licensing-and-vertical-description-param-vertical-movies": "Movies",
   "global-footer-full-site-link": "View Full Site",
-  "global-footer-mobile-site-link": "View Mobile Site"
+  "global-footer-mobile-site-link": "View Mobile Site",
+  "global-navigation-anon-my-account": "My Account",
+  "global-navigation-anon-register": "Register",
+  "global-navigation-anon-register-description": "Don't have an account?",
+  "global-navigation-anon-sign-in": "Sign In",
+  "global-navigation-create-wiki-link-start-wikia": "Start a Wiki",
+  "global-navigation-fandom-overview-link-vertical-games": "Games",
+  "global-navigation-fandom-overview-link-vertical-movies": "Movies",
+  "global-navigation-fandom-overview-link-vertical-tv": "TV",
+  "global-navigation-notifications-title": "Notifications",
+  "global-navigation-search-cancel": "Cancel",
+  "global-navigation-search-placeholder-fandom": "Search Stories and Wikis...",
+  "global-navigation-search-placeholder-in-wiki": "Search __sitename__",
```

Comment:
Are we sure about this one? We finish other active placeholders with three dots. So we we have an exception here.
The same pattern is followed on production.